Project-I-Social-Card/social-card/src/components/FooterComponents/Footer.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import './Footer.css';
+import FooterIcons from './FooterIcons';
+
+const Footer = () => {
+
+return (
+<div className="footer">
```

Comment:
I'd move this wrapper div inside the `footer` component, so it is self-contained and easier to be reused else where